src/main/java/org/candlepin/resource/ConsumerResource.java

```diff
@@ -2550,6 +2548,15 @@ public Response bind(
         ConsumerType ctype = this.consumerTypeCurator.getConsumerType(consumer);
         log.debug("Consumer (post verify): {}", consumer);
 
+        // Not allow to attach ent to consumer in SCA mode(Only manifest consumers are allowed)
+        boolean scaEnabled = consumer.getOwner().isUsingSimpleContentAccess();
+        if (scaEnabled && !ctype.isManifest()) {
+            ExceptionMessage message = new ExceptionMessage(
+                i18n.tr("Ignoring request to attach entitlements. It is disabled for org: {0} because of " +
```

Comment:
Let's clean up this error message, and try to avoid splitting the string across multiple lines, as that makes searching for it harder.

Something to the effect of: "Bind operations are disabled in organizations using simple content access". Maybe "ignored" is better in this context since it's not a failure?

May be valuable to reach out to PMs for recommendations on how this should be expressed to end users.

Reply:
To add to this: by now, most components that used to call this API (Satellite, RHSM-Web, RHSM-API, subscription-manager in RHEL8+) have been patched to simply not call it when in SCA mode. The only thing we need to test is old subscription-manager clients (RHEL7/RHEL6). What is expected is, the operation should:
- not result in an unhandled stacktrace error
- should result in a descriptive message to the user
I don't know how subman in RHEL7/6 works, so I'm not sure if sending back a 200 status with an exception message will work. Perhaps it could be a 4XX error code that would work

Reply:
i did not throw error because this is written in bugzilla: `(We do not want to throw an error in order to make sure older, unpatched clients do not break)`

Reply:
I think I wrote that BZ, and I agree its not very clear. The goal is for unpatched clients (RHEL7 and 6) to not have unhandled errors that break them. As long as the client is able to handle this by, for example, showing user-friendly error, then we are fine

Reply:
have you tested with a RHEL7 or 6 client? how does it look from that perspective?

Reply:
Updated the error message, opting to use "Attach" instead of "Bind" because the customer is using subman attach. The response now returns a 400 status code instead of 200. Verified on RHEL 6 to ensure this change does not break the subman client. The client will display the updated error message we provided and still conclude with a zero exit code.